docs/en/observability/logs-stream.asciidoc

```diff
@@ -319,17 +339,26 @@ After creating your ingest pipeline, create an index template to point your log
 ----
 PUT _index_template/logs-example-default-template
 {
-  "index_patterns": [ "logs-example-default" ],
+  "index_patterns": [ "logs-example-*" ],
   "data_stream": { },
   "priority": 500,
   "template": {
     "settings": {
-      "index.default_pipeline":"logs-example-default"
+      "index.default_pipeline":"logs-example"
     }
-  }
+  },
+  "composed_of": [
```

Comment:
@mdbirnstiehl In a follow up PR, we should explain in details what these are doing. I would like to quickly sync with you on this to discuss how we describe this best. A quick summary to get it storated:

Reply:
Thanks @eyalkoren . Not sure what you mean by dynamic vs perma links?

Reply:
A link to the corresponding template file in main (e.g. [`logs-mappings.json`](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/core/template-resources/src/main/resources/logs-mappings.json)) would be useful, but may break docs build if changed/moved (just like [happened recently](https://github.com/elastic/elasticsearch/pull/97763)).
A link to a permanent URL (e.g. [`logs-mappings.json`](https://github.com/elastic/elasticsearch/blob/1955bd8ad421e4acc37dbb4e948c5ff80fce4131/x-pack/plugin/core/template-resources/src/main/resources/logs-mappings.json)) will not be very useful.

Reply:
I really like the idea of being able to link to it or copy it over and update it from time to time. Having the templates in the docs directly will also allow to explain these better. Maybe we need a page where these are explained, a dedicate page? If we have this, how would we keep it up-to-date there? Seems like one of these common issues on how to keep docs and code in sync.

Reply:
If we come up with a method to keep docs up-to-date somehow, maybe we can use permanent links and keep them up to date.

Reply:
@mdbirnstiehl @bmorelli25 Has something like this been done in the past?

Reply:
@bmorelli25 can answer maybe better than me if this fits the situation. BI think in the APM realm some files that have ties to docs require a review from obs-docs before merging to prevent the file from being updated without the corresponding docs. Not sure if we could do something like that in this instance.